OracleWebLogic/dockerfiles/12.2.1.2/container-scripts/createAndStartEmptyDomain.sh

```diff
@@ -2,31 +2,6 @@
 #
 # Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
 #
-########### SIGTERM handler ############
-function _term() {
-   echo "Stopping container."
-   echo "SIGTERM received, shutting down database!"
-   sqlplus / as sysdba <<EOF
-   shutdown immediate;
-EOF
-   lsnrctl stop
-}
-
-########### SIGKILL handler ############
```

Comment:
We still need a `SIGKILL` handler, because the `createAndStartEmptyDomain.sh` starts WebLogic and then tails the `AdminServer.log` file. Sending a CTRL+C inside the container (or telling it to stop) won't actually shutdown WebLogic.

You need to work out the correct way to abort a WebLogic instance and then call that in `SIGKILL`.